client/my-sites/stats/hooks/use-should-gate-stats.ts

```diff
@@ -115,6 +116,7 @@ const gatedStats = [
 	STATS_FEATURE_INTERVAL_DROPDOWN_WEEK,
 	STATS_FEATURE_INTERVAL_DROPDOWN_MONTH,
 	STATS_FEATURE_INTERVAL_DROPDOWN_YEAR,
+	STATS_FEATURE_SUMMARY_LINKS_30_DAYS,
```

Comment:
```suggestion
	STATS_FEATURE_SUMMARY_LINKS_7_DAYS,
	STATS_FEATURE_SUMMARY_LINKS_30_DAYS,
```
We may also need to gate even `7 days` as the whole Stats modules are all gated on the Traffic page for Free sites created after the `stats/paid-wpcom-v3` release.

<img width="1417" alt="截圖 2024-12-12 晚上10 39 34" src="https://github.com/user-attachments/assets/c5836bba-4612-4a8e-8e50-537cce67fcce" />

Reply:
Makes sense to me, 

v2
![Screenshot 2024-12-13 at 12-47-38 Jetpack Stats ‹ Site Title — WordPress com](https://github.com/user-attachments/assets/552953ac-b026-4f99-abb8-febdac5b06db)

v3
![Screenshot 2024-12-13 at 12-48-03 Site Title — WordPress com](https://github.com/user-attachments/assets/fc50b309-df1c-41bc-aadc-bcfec59acf71)